src/game/logic/ai/aigroup.cpp

```diff
@@ -1686,6 +1694,7 @@ void AIGroup::Group_Tighten_To_Position(const Coord3D *target_pos, bool append,
         }
     }
 
+    // Update the positions of the members in the Pathfinder
```

Comment:
I think these kind of in-function comments are too much. The code here is rather self-explanatory. The comment outside the function is good enough.

In-function code comments are useful when something not so obvious needs explanation - or in our case, when we deviate from original functionality. However, even that can be documented by code, for example by putting new code into a unique namespace, or by guarding code inside a pre-processor macro.